src/test/analyzer_test.ts

```diff
@@ -168,6 +168,21 @@ suite('Analyzer', () => {
           ['polymer.html', 'foo.js', 'foo.css']);
     });
 
+    test('polymer css import scanner', async() => {
+      let contents = `<html><head>
+          <link rel="import" type="css" href="bar.css">
```

Comment:
It turns out that there's already magic in analyzer preventing the scanner from diving into generic `<template>`.  I haven't tried with `<template is="dom-bind">` but I did modify test to demonstrate the analzyer's behavior vis-a-vis `<dom-module>` and `<template>`.